waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -861,8 +873,31 @@ proc mountRlnRelay*(node: WakuNode, conf: WakuNodeConf) {.raises: [Defect, Value
       let memKeyPair = keyPair.toMembershipKeyPairs()[0]
       # mount the rln relay protocol in the on-chain/dynamic mode
       waitFor node.mountRlnRelayDynamic(memContractAddr = ethMemContractAddress, ethClientAddr = ethClientAddr, memKeyPair = some(memKeyPair), memIndex = some(rlnRelayIndex), ethAccAddr = ethAccountAddr, ethAccountPrivKey = ethAccountPrivKey, pubsubTopic = conf.rlnRelayPubsubTopic, contentTopic = conf.rlnRelayContentTopic)
+    elif fileExists("keyPair.txt") and fileExists("rlnIndex.txt"):
+      info "keyPair and rlnIndex files exist"
+      # This will read the entire file into the string entireFile
+      let entireKeyPairFile = readFile("keyPair.txt")
+      # echo entireKeyPairFile  # prints the entire file
+
+      let jsonObjectKeyPair = parseJson(entireKeyPairFile)
+      let deserializedKeyPair = to(jsonObjectKeyPair, MembershipKeyPair)
+
+      info "Deserialized key pair"
+      echo deserializedKeyPair # prints the deserialized key pair
+
+      # This will read the entire file into the string entireFile
```

Comment:
Imo, this comment is not necessary. See above.

Reply:
Suggestion Incorporated